config/index.js

```diff
@@ -7,8 +7,10 @@ module.exports = {
     port: process.env.PORT || 8001,
     logs: {
       console: true,
-      app: false,
-      error: false
+      app: 'logs/lev-web-ui.log',
```

Comment:
I think potentially the file could be removed with the date rotation being set to true but the documentation isn't clear. If it works, fine.

Reply:
yup, may be we just need to keep an eye on it. if the files are removed then need something in place for it

Reply:
At what point is the log file expected to rotate?